### ISSUE TITLE
Add variables for statute amendments

### ISSUE THREAD

**Issue by georgd** [citation-style-language/schema#339]
> Ok. You know way more about legal citations than I. What would be needed for more complete legal citation support in CSL independent of CSLm? What are the core feature that we would have to adopt?

_Originally posted by @denismaier in https://github.com/citation-style-language/schema/issues/320#issuecomment-660364055_

Amending statutary law is a regular process, so a statute is changing over time. Thus, for correct citation it is necessary to state the version one is referring to. What’s necessary might vary. It could be the amended date alone but it could also require the complete citation of the issue of the official gazette where the amendment was published.

Current practice is to use the `history` field to enter all the required information as a formatted string which might lead to style specific entry. To avoid the latter, four additional variables are necessary:

`amended-date`, `amended-container-title`, `amended-volume` and `amended-number`.

Actual case (Austrian law, sanctioned in 1986 and published in BGBl no. 472, with amendments published up to 2008 in BGBl first series, no. 117), cited according to two different styles:
> Schulunterrichtsgesetz, BGBl 472/1986 idF BGBl I 117/2008
vs.
> Schulunterrichtsgesetz, BGBl 1986/472 idF BGBl I 2008/117

**Comment by denismaier** [comment]
How is that different from reprint information? Or could that be handled with `original-` variables or the like? I'm just wondering whether there could be a more general mechanism to handle this use case?

**Comment by georgd** [comment]
> How is that different from reprint information?

The semantics don’t fit at all, of course — a reprint doesn’t change the contents, whereas an amendment explicitly does.

> Or could that be handled with `original-` variables or the like? I'm just wondering whether there could be a more general mechanism to handle this use case?

Again, the semantics :). At the moment I don’t even see a complete set of variables to which one could map these fields. There are `original-date` and `original-title` (semantically not a container title) but no equivalents for volume and number.

In general I don’t see an issue in mapping to a set of variables that is not used with statutes (we have been doing this all the time :)) — as long as there are graphical applications that hide the real variable names from the majority of the users.

**Comment by denismaier** [comment]
> At the moment I don’t even see a complete set of variables to which one could map these fields. There are `original-date` and `original-title` (semantically not a container title) but no equivalents for volume and number.

In 1.1. we'll change `original` to an object that can digest just every other regular variable. That doesn't mean it does work, just that every field should be available eventually.

**Comment by georgd** [comment]
> In 1.1. we'll change `original` to an object that can digest just every other regular variable. That doesn't mean it does work, just that every field should be available eventually.

Right, I forgot already and didn’t read the relevant schema file. This could indeed work. However, I’m not sure about the best representation when going with a related structure (are they now embedded or are the related objects independent?) — if `original` or `reviewed` were the better fit.

The amendments are defined by a separate amending law that (much like a diff file) defines which words, sentences or paragraphs need to be cancelled, exchanged or added. And the same amending law can amend multiple original laws. So, the same amended-publication data can be related to various original statutes. Also, a scholar might want to cite the amending law itself.

I need to think a bit more about this option.

**Comment by denismaier** [comment]
I'd say `original` would be better. But we could also think about a `amended` object if that better fits this purpose. Perhaps there are other cases where such revisions are needed. @bwiernik @bdarcus ?

**Comment by bdarcus** [comment]
I've not yet read this thread carefully, but ...

If you cite the amended document, then original seems appropriate to me.

I can't see the use case for an amended relation; if we needed to flp
the relation around like that, we'd want a more general relation?

**Comment by georgd** [comment]
From a semantic point of view, `original` has its logic. However, the statutes are always cited by their original version (see examples above). The amendment data don't even appear in subsequent citations except if two different versions of the same statute are cited. The short citation for the above looks like this:

> SchUG 1986

Only if two different amended versions cited:

> SchUG 1986 idF BGBl I 2008/117

(Side note: this kind of disambiguation needs implementation as well but I don't think it's necessary in general purpose styles so I'd restrict it to CSLm for the moment)

So, the original data have a more prominent position than the amendment data whence I fear going with `original` might pose usability issues? Looking at Zotero however, this could also simplify identifying the desired version from the citation dialog (currently it's like flying blind...).

**Comment by denismaier** [comment]
That's why I asked about reprints. If that's misleading, what about `revision`?

**Comment by bdarcus** [comment]
Dublin Core has version and versionOf as top-level relations.

Not suggesting that necessarily, but FYI.

**Comment by bwiernik** [comment]
@georgd There is no `history` field. I assume you mean `references`? 

For any legal citation questions, we should first consult what CSLm is doing. Frank and I discussed the `@related` issue in legal citation at some length when we sought input on `@related` a few weeks back. There are some important differences in these work with legal citations versus normally that we should consider. Let me look up a few things and write some analysis.

**Comment by bwiernik** [comment]
In legal citation, the citation is typically to the original version, with amendments, repeals, affirmations, overturning or conflicting decisions, etc. cited using a structure like `related`. This is referred to as "parallel citation". See [R12.3.Parallel Citation in State Court Documents at this page](https://juris-m.github.io/indigobook/) for an example. Occasionally, these are done in the reverse order. The nature of the relationship is rarely indicated (see the Indigo Book example I linked to or @georgd's example, where it's just a delimiter). In my conversations with Frank, he describes the order and whether they are included as being a choice the writer makes in constructing their argument. 

With that in mind, in CSLm, legal parallel (i.e., `@related`) citations are handled by a general `seeAlso` relationship, where the CSL data with `seeAlso` relationships providing a list of related item IDs. When items with `seeAlso` relationships are cited adjacent to each other, they are combined following the parallel citation rules specified in the legal citation style. See [here for details](https://citeproc-js.readthedocs.io/en/latest/csl-m/#parallel-citation-and-suppressing-repetition). These combination rules vary across jurisdictions and specific cases. My understanding watching Frank's work on the Indigo Book integration and modular comparative law formatting is that this is probably the most complex part of doing legal citation.

@fbennett Is the specific nature of the relationships between parallel items ever indicated? If so, how is that supplied in the data?

The relationship of an amendment to a statute is part of the same general family as other legal parallel citations. I think we should leave `legalSeeAlso` type relationships together and also get into them in vanilla CSL. This is one of the major complexities of legal citation.

**Comment by georgd** [comment]
> @georgd There is no `history` field. I assume you mean `references`?

Right, it’s `references` in CSL. `history` is the corresponding Zotero field label (which I don’t understand in this context).

> For any legal citation questions, we should first consult what CSLm is doing. Frank and I discussed the `@related` issue in legal citation at some length when we sought input on `@related` a few weeks back.

There’s currently nothing about amended statutes in CSLm as far as I can tell. 

You’re citing *case law* parallel citations which I’d consider the most complex part of legal citation as well, especially the peculiarities in common law systems like the US or UK. CSLm meanwhile seems to handle it very well and it isn’t far away from handling German and Austrian specifics as well.

> ... See [here for details](https://citeproc-js.readthedocs.io/en/latest/csl-m/#parallel-citation-and-suppressing-repetition). 

This is outdated. Frank reimplemented the parallel mechanism not long ago with a different set of attributes. This blog post explains them: https://juris-m.github.io/posts/2020-07-20.html

> The relationship of an amendment to a statute is part of the same general family as other legal parallel citations.

I’m not sure if this can be accurate. Besides the “... amendments, repeals, affirmations, overturning or conflicting decisions, etc.”, parallel citations are particularly necessary because court decisions are often published in multiple sources. In the US there seems to be a financial dimension to this. What’s more, decisions published in journals often come with a commentary. So, citing all the parallel sources is considered good practice. And citing the history of subsequent decisions on one single case or a group of similar cases is important to understand the reasoning of the verdict.

Statute amendments, on the other hand, are much more like versioning. When one cites an amended statute one cites exactly that version, without considering what the previous version or the subsequent version says.

> I think we should leave `legalSeeAlso` type relationships together and also get into them in vanilla CSL. This is one of the major complexities of legal citation.

If you say so, I’m all for it :)

**Comment by bwiernik** [comment]
I meant _not_ get into it in vanilla CSL sorry

**Comment by bwiernik** [comment]
> This blog post explains them: https://juris-m.github.io/posts/2020-07-20.html

Thanks

**Comment by bwiernik** [comment]
With respect to citing statutes history, I am reluctant to introduce a system that would object even more complexity into the parallel legal citations, so I would like to hear from @fbennett.

**Comment by georgd** [comment]
> I meant _not_ get into it in vanilla CSL sorry 

Haha, I already wondered what happened — that's ok for me :)

**Comment by georgd** [comment]
> With respect to citing statutes history, I am reluctant to introduce a system that would object even more complexity into the parallel legal citations, so I would like to hear from @fbennett. 

I also wouldn't and I don't think it fits in there.

**Comment by bwiernik** [comment]
For example, when I cite the US Civil Rights Act of 1964, I am only intending to cite that act, not its amendment by the Civil Rights Act of 1990, unless I am specifically referring to the provisions that were amended. That seems to align exactly with the structure of parallel case citations where an author may or may not cite a related item depending on the purpose.
